docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/security-and-authentication.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Security and Authentication
+description: This article describes the authorization mechanism used in Spryker, as well as user scopes, database tables, and extension points.
+last_updated: Oct 24, 2022
+template: glue-api-storefront-guide-template
```

Comment:
Where do we have this template?

Reply:
_templates/glue-api-storefront-guide-template.md